server/cmd/mmctl/commands/post.go

```diff
@@ -100,13 +105,36 @@ func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	channelID := ""
+	if strings.HasPrefix(args[0], "@"){
+
+	me,_, err := c.GetMe(context.TODO(), "")
+	if err!=nil{
+		return fmt.Errorf("could not get current user: %s", err.Error())
+	}
+
+	userArgString := strings.Split(args[0], "@")[1]
+	user := getUserFromUserArg(c,userArgString)
+	if user == nil {
+    return fmt.Errorf("unable to find user %q", userArgString)
+  }
+
+	directChannel,_,err := c.CreateDirectChannel(context.TODO(),me.Id,user.Id)
+	if err!=nil{
+    return fmt.Errorf("could not create direct channel: %s", err.Error())
+  }
+	channelID = directChannel.Id
+	}else{
```

Comment:
The formatting looks wrong. Are you sure that the file is `gofmt`ed?

Reply:
Not yet @hanzei

Reply:
Hello @hanzei 
I have added a test case to test for user not found. I don't know if this additional test case will make the testSuit complete.

Reply:
I am also facing this problem when I try to login via `mmctl auth login` with the error below

```
Error: could not initiate client: Post "http://localhost:8086/api/v4/users/login": dial tcp 127.0.0.1:8086: connect: connection refused
```